code/game/objects/structures/spirit_board.dm

```diff
@@ -108,4 +108,4 @@
 			to_chat(ghost, span_warning("There aren't enough people around to use [src]!"))
 			return FALSE
 
-	return TRUE
+	return TRUE
```

Comment:
Add a newline here, linters aren't happy.

Reply:
Ill just fix this for them